.github/workflows/deploy-lambda.yml

```diff
@@ -5,7 +5,7 @@ name: CI
 on:
   push:
     branches:
-      - "serverless"
```

Comment:
This just for testing out this branch. I'll switch it back to `serverless` before merging